### DISCUSSION TITLE
Confused about converting account to organization..??

### DISCUSSION THREAD

**Discussion by angelleye** [community/community#23263] [New to GitHub]
<p>I am currently registered with GitHub Pro, and I would like to switch to GitHub Organizations.</p>
<p>I’ve read all the conversion guides for this, and I see my options about converting the account vs. creating a new organization and then trasferring the repos.  I would like to do the <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/converting-a-user-into-an-organization" rel="nofollow noopener">conversion</a>.</p>
<p>I’m prepared for this, but when I go into the conversion steps, I end up with this option:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/46f21734fe42d7d4a6fecc3f91603aebad705e17.png" data-download-href="/uploads/short-url/a7C6bu4WIEoXrFQljPWSgnlSBYX.png?dl=1" title="github-conversion.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/629241/181075961-963b33b6-acce-4729-af7d-a78df2aa112c.png" alt="github-conversion.png" data-base62-sha1="a7C6bu4WIEoXrFQljPWSgnlSBYX" width="641" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/46f21734fe42d7d4a6fecc3f91603aebad705e17_2_641x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/46f21734fe42d7d4a6fecc3f91603aebad705e17_2_961x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/46f21734fe42d7d4a6fecc3f91603aebad705e17.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/46f21734fe42d7d4a6fecc3f91603aebad705e17_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github-conversion.png</span><span class="informations">1068×832 48.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>My natural reaction here is to choose what I have, because this is indeed for my business.  However, the note below that scares me a bit.  What does it mean that “not angelleye” will control this organization.  What account controls it then?</p>
<p>With both options, it still has me select the organization owner, which is my new personal account I have created.</p>
<p>So I’m just a bit confused on what I should be selecting here.  Any information on this would be greatly appreciated.  Thanks!</p>

**Comment by angelleye** [comment]
<p>Nobody can help me out here?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/angelleye">@angelleye</a>,</p>
<p>Thank you for being here! Designating your personal account as the owner is the correct step, since it is your organization and for security purposes, you would be the owner. Please ensure that you have another user account ready <em>(as I believe you already do)</em> to accept ownership of the account following conversion and - once that is done - that other user account would then be needed to access the converted account; as an organization cannot be access directly.</p>

**Reply by laultman** [reply]
My company has an organizational account that is a paid account. I personally have had an account for years and don't want to lose all my past history. I now have two accounts, my personal and business. I would like to merge them into my business account while keeping my history. Both my accounts have paid CoPilot option. I log in to my VS using my business and it is associated with my CoPilot while 99% of the time I use my personal for issues and responses. Is there a way to merge my personal into my business?

**Comment by ghost** [comment]
<p>Hello <a class="mention" href="/angelleye">@angelleye</a>,</p>
<p>That option determines ownership of the organization with respect to our Terms of Service.</p>
<p>If you select the " <strong>My personal account"</strong>  option, the organization is bound to our <a href="https://help.github.com/en/github/site-policy/github-terms-of-service" rel="nofollow noopener">Standard Terms of Service</a> with you as the owner. Selecting the <strong>“A business or institution”</strong> option places the organization on our <a href="https://help.github.com/en/github/site-policy/github-corporate-terms-of-service" rel="nofollow noopener">Corporate Terms of Service</a> instead, with ownership held by an entity such as a company, non-profit, or group. Corporate Terms provide greater flexibility with regard to account ownership disputes and transfers, which can be beneficial in cases where an organization owner leaves a business or otherwise becomes unresponsive.</p>

**Comment by angelleye** [comment]
<p>Thanks for the reply.</p>
<p>So if I select to set this up as corporate, because my company is indeed an S-Corp, would it still be showing up under /angelleye or does that get changed?  That’s the part that’s confusing me, because it says the company “not angelleye” would control it.</p>
<p>Thanks!</p>

**Comment by ghost** [comment, marked as answer]
<p>Your personal account will still have <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization" rel="nofollow noopener">owner permissions</a> for the organization, so the way in which you <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-your-organization-dashboard#accessing-your-organization-dashboard" rel="nofollow noopener">access the organization’s content</a> remains the same, regardless of who owns the organization from a Terms of Service perspective.</p>

**Comment by becke-ch-s0-v1** [comment]
Hi @AndreaGriffiths11, @angelleye and @ghost , 
thank you for this post which I think should be part of the official documentation https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch and/or https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/converting-a-user-into-an-organization
Because I could nowhere in the official documentation find a description and "consequences" of these choices.
I'm right now in the same process of converting a personal into an organization account.
Besides the "Corporate Terms of Service" is there any difference in the registration process? Do I need somehow to bring proof of me owning the organization and how? Does this choice has any financial impact?
Thank you
